content/docs/user-guide/components/reference/gradients/altitude-gradient.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
I'm not requesting you update the screen shots in this PR, just a note for the future. If the interface has some sort of preview window, we should try to get a screenshot with some data in the preview.

Reply:
Yeah, I wanted to (for previous PRs), my concern was that you need specific component configurations that could hide properties.  That's not a problem for every component though.  Multiple previews might make sense, the generic one for properties definition (with preview content where possible) and then any additional previews.  For example, Fastnoise could use use ~half a dozen previews.